### DISCUSSION TITLE
Option to import songs and skins from an already existing osu lazer folder

### DISCUSSION THREAD

**Discussion by Smorty100** [ppy/osu#14535] [Ideas] [answered]
We already have the option to import songs and skins and even scores from osu stable. But we don't have the option to transfer osu lazer files yet. It would be really useful, if we would beable to wither direct osu lazer to a directory, which will be the new directory in which songs and skins will be saved, or to give us the option to direct lazer to an old lazer folder and import all which is in it into the new lazer folder. So basically the function of importing stuff from stable... but instead importing from stable, one can import from lazer.

**Comment by peppy** [comment, marked as answer]
Are you wanting to *combine* two installs, rather than just  migrate to a new PC? If you need to migrate, you can just copy the files.

**Reply by Smorty100** [reply]
Oh... yup, you're right. Didn't think it would be that simple.

**Reply by ozornayashoujo** [reply]
Hey, I know this is an old thread, but I tried "just copying the files", that being the Local and Roaming AppData folders, but the game won't function properly. Specifically: the beatmap list is there, the replay files work fine, but the maps have no sound and the skins are also all wrong. I'm at a loss because I was hoping I could just "move" my lazer installation to my new computer.
To be specific, here's an example error message:
_2023-06-01 22:48:26 [verbose]: ⚠️ Beatmap failed to load (file audio.mp3 not found on disk at expected location c\ce\ce9a860ef4f8d7a0f447e3480295864dde361da2a37451faed63a6d97f5d4748)._

**Reply by peppy** [reply]
Sounds like you didn't copy everything. If replays are working, you at least copied *some* files, so maybe try again and make sure they all exist.

**Reply by ozornayashoujo** [reply]
> Sounds like you didn't copy everything. If replays are working, you at least copied _some_ files, so maybe try again and make sure they all exist.

Good call. I checked for any other files/folders with "osu" in the filename and found "SquirrelClowdTemp". I would've never guessed it, but once I copied that the game ran perfectly well with no missing files/wrong addresses so far. Thanks for the help.

**Reply by peppy** [reply]
`SquirrelClowdTemp` is definitely not what fixed your issue. You probably tried running osu! while the files were still copying. Anyways, glad you could get it resolved.
